Client/src/Pages/Monitors/CreateMonitor/index.jsx

```diff
@@ -40,6 +42,21 @@ const CreateMonitor = () => {
   const [https, setHttps] = useState(true);
   const [errors, setErrors] = useState({});
 
+  useEffect(() => {
+    if(monitorId){
+      const data = monitors.find((monitor) => monitor._id === monitorId);
+      if (!data) {
+        console.error("Error fetching monitor of id: " + monitorId);
+        navigate("/not-found");
+      }
+      const {url, name, ...rest} = data;
```

Comment:
I think we would want the `url` and `name` included in the new monitor as we are creating a copy, no need to exclude these.

Perhaps consider appending `" (Clone)"` or somethig similar to the name to signify that this is a copy